nodes.adoc

```diff
@@ -0,0 +1,1163 @@
+
+[source,terminal]
+----
+$ oc create configmap -n openshift-config --from-file=policy.cfg <configmap-name> 
+----
+
+
+[source,terminal]
+----
+$ oc create configmap -n openshift-config --from-file=policy.cfg scheduler-policy
+----
+
+
+[source,terminal]
+----
+$ oc patch Scheduler cluster --type='merge' -p '{"spec":{"policy":{"name":"<configmap-name>"}}}' --type=merge 
+----
+
+
+[source,terminal]
+----
+$ oc patch Scheduler cluster --type='merge' -p '{"spec":{"policy":{"name":"scheduler-policy"}}}' --type=merge
+----
+
+
+[source,terminal]
+----
+$ oc edit configmap <configmap-name>  -n openshift-config
+----
+
+
+[source,terminal]
+----
+$ oc edit configmap scheduler-policy -n openshift-config
+----
+
+
+[source,terminal]
+----
+$ oc delete configmap -n openshift-config <name>
+----
+
+
+[source,terminal]
+----
+$ oc delete configmap -n openshift-config  scheduler-policy
+----
+
+
+[source,terminal]
+----
+$ oc create configmap -n openshift-config --from-file=policy.cfg <configmap-name> 
+----
+
+
+[source,terminal]
+----
+$ oc create configmap -n openshift-config --from-file=policy.cfg scheduler-policy
+----
+
+## Controlling pod placement on nodes using node affinity rules
+
+[source,terminal]
+----
+$ oc label node node1 e2e-az-name=e2e-az1
+----
+
+
+[source,terminal]
+----
+$ oc create -f e2e-az2.yaml
+----
+
+
+[source,terminal]
+----
+$ oc label node node1 e2e-az-name=e2e-az3
+----
+
+
+[source,terminal]
+----
+$ oc create -f e2e-az3.yaml
+----
+
+
+[source,terminal]
+----
+$ oc label node node1 zone=us
+----
+
+
+[source,terminal]
+----
+$ oc label node node1 zone=emea
+----
+
+## Placing pods on specific nodes using node selectors
+
+[source,terminal]
+----
+$ oc patch MachineSet <name> --type='json' -p='[{"op":"add","path":"/spec/template/spec/metadata/labels", "value":{"<key>"="<value>","<key>"="<value>"}}]'  -n openshift-machine-api
+----
+
+
+[source,terminal]
+----
+$ oc patch MachineSet abc612-msrtw-worker-us-east-1c  --type='json' -p='[{"op":"add","path":"/spec/template/spec/metadata/labels", "value":{"type":"user-node","region":"east"}}]'  -n openshift-machine-api
+----
+
+
+[source,terminal]
+----
+$ oc edit MachineSet abc612-msrtw-worker-us-east-1c -n openshift-machine-api
+----
+
+
+[source,terminal]
+----
+$ oc label nodes <name> <key>=<value>
+----
+
+
+[source,terminal]
+----
+$ oc label nodes ip-10-0-142-25.ec2.internal type=user-node region=east
+----
+
+
+[source,terminal]
+----
+$ oc edit scheduler cluster
+----
+
+
+[source,terminal]
+----
+$ oc patch MachineSet <name> --type='json' -p='[{"op":"add","path":"/spec/template/spec/metadata/labels", "value":{"<key>"="<value>","<key>"="<value>"}}]'  -n openshift-machine-api 
+----
+
+
+[source,terminal]
+----
+$ oc patch MachineSet ci-ln-l8nry52-f76d1-hl7m7-worker-c --type='json' -p='[{"op":"add","path":"/spec/template/spec/metadata/labels", "value":{"type":"user-node","region":"east"}}]'  -n openshift-machine-api
+----
```

Comment:
Requires a declarative alternative

Reply:
This is an example of the previous command, in the same step. Added on declarative for both.